contentgrid-spring-boot-autoconfigure/src/main/java/com/contentgrid/spring/boot/autoconfigure/automation/ContentGridAutomationAutoConfiguration.java

```diff
@@ -1,35 +1,18 @@
 package com.contentgrid.spring.boot.autoconfigure.automation;
 
-import com.contentgrid.automations.rest.AutomationAnnotationRepresentationModelAssembler;
-import com.contentgrid.automations.rest.AutomationRepresentationModelAssembler;
-import com.contentgrid.automations.rest.AutomationsRestController;
-import com.contentgrid.thunx.predicates.model.ThunkExpression;
-import com.contentgrid.thunx.spring.data.context.AbacContextSupplier;
-import org.springframework.beans.factory.annotation.Autowired;
+import com.contentgrid.automations.ContentGridAutomationsConfiguration;
+import com.contentgrid.spring.boot.autoconfigure.data.web.ContentGridSpringDataRestAutoConfiguration;
 import org.springframework.boot.autoconfigure.AutoConfiguration;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Import;
 
-@AutoConfiguration
+@AutoConfiguration(
+        after = ContentGridSpringDataRestAutoConfiguration.class
+)
 @ConditionalOnWebApplication
-@ConditionalOnClass({ AutomationsRestController.class, AbacContextSupplier.class, ThunkExpression.class })
```

Comment:
Are `AbacContextSupplier` and `ThunkExpression` automatically on the classpath when `ContentGridAutomationsConfiguration` is on the classpath?

Reply:
I might have been a bit too optimistic about that